datastream-server/src/main/java/com/linkedin/datastream/server/CoordinatorEvent.java

```diff
@@ -8,6 +8,9 @@
 
 /**
  * Represents different event types inside {@link Coordinator}.
+ *
+ * CoordinatorEvent will be deduped in the event queue based on the event type
```

Comment:
Could you please add description for what the bug was?

Reply:
added a link